examples/experimental/dagster-airlift/airflow_requirements.txt

```diff
@@ -0,0 +1,11 @@
+# Requirements for python versions under 3.12.
+apache-airflow>=2.0.0; python_version < "3.12"
+pendulum>=2.0.0,<3.0.0; python_version < "3.12"
+# Requirements for python versions 3.12 and above.
+apache-airflow>=2.9.0; python_version >= "3.12"
+pendulum >= 3.0.0; python_version >= "3.12"
+# Flask-session 0.6 is incompatible with certain airflow-provided test
+# utilities.
+flask-session<0.6.0
+# https://github.com/apache/airflow/issues/35234
+connexion<3.0.0
```

Comment:
Why put these in a file?

Reply:
Vestigial from when I thought I would need to do something wacky to install airflow in the tutorial. You're right that there's no need